resources/dicts/patrols/general/medcat.json

```diff
@@ -212,5 +212,83 @@
     "max_cats": 6,
     "antagonize_text": "app1 feels hounded, never given a moment's peace, and they hiss at the StarClan cat. The ghost vanishes like a puff of fog in the morning sun, hopefully not to return.",
     "antagonize_fail_text": "app1 feels hounded, never given a moment's peace, and they hiss at the StarClan cat. But when they turn around, there's only a rather upset looking clanmate, there to fetch them to camp to help with an emergency."
+},
+{
+    "patrol_id": "gen_med_medwar_natureofstarclan",
+    "biome": "Any",
+    "season": "Any",
+    "tags": ["med_cat", "warrior", "random_herbs", "no_app"],
+    "intro_text": "During a patrol looking for herbs, p_l notices that r_c seems more quiet than usual. When prompted, r_c slowly asks if p_l could teach them more about the nature of StarClan.",
+    "decline_text": "p_l opens their mouth to answer, but the cry of another patrol looking for them shuts down the conversation. p_l races away towards the scene leaving r_c behind.",
+    "chance_of_success": 60,
+    "exp": 10,
+    "success_text": [
+        "p_l eventually relents and begins teaching r_c more about StarClan and the nature of prophecies. Expecting boredom, p_l was shocked when they saw wonder in r_c's face. The two spend the rest of the patrol pondering the nature of their ancestors together.",
+        "p_l eventually relents and begins teaching r_c more about StarClan and the nature of prophecies. r_c takes to it quickly, marveling at their ancestors. They attempt to bub p_l about any recent prophecies, but p_l only laughs and gets the patrol back on track for herbs.",
+        "p_l had seen before how connected s_c was to StarClan, and began telling them more than they would have told any other Clan cat about their ancestors. p_l and s_c spend much of their patrol gathering herbs and discussing the hard-hitting questions on prophecies and StarClan's gifts."
```

Comment:
unfortunately, with how stat cats and random cats work now, the stat cat and random cat cannot be the same cat.  so you'll need to rewrite this so that the patrol leader is the stat cat instead.  I'll go make sure the patrol bible is up to date with that info